orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -234,6 +239,8 @@ def _get_session_config(self):
         return session_options, providers, provider_options
 
     def _export_model(self, *inputs, **kwargs) -> bool:
+        self._time_tracker.start(_logger.TimeTrackerPhase.EXPORT)
```

Comment:
Shall we use a decorator instead? That way we can avoid having to update the function body definition each time we want to add a timer to it.

Reply:
I did not find an elegant way to add a decorator for class member function, so I used a context manager to achieve this, let me know if you have a better solution. :)

Reply:
```py

import time

class TrackIt:
    def __init__(self, tracker_type):
        self._tracker_type = tracker_type

    def __call__(self, f):
        def wrapped(*args, **kwargs):
            start_time = time.perf_counter()
            ret = f(*args, **kwargs)
            end_time = time.perf_counter()
            print(f"Function {f.__name__} took {end_time - start_time} seconds. tracker_type is {self._tracker_type}")
            return ret

        return wrapped


class A:
    def __init__(self):
        pass

    @TrackIt("add_a_thousand_times")
    def add_a_thousand_times(self, a, b):
        for _ in range(1000):
            a += b

        return a

a = A()
c = a.add_a_thousand_times(1, 2)
print(c)


```

Reply:
Thanks a lot!